monash-human-power/MHP_DAS
# Add ANT+ service

## Description

This PR adds a systemd service for the ANT+ script, so that it can easily be run on the bike.

I also updated some READMEs and moved the services to a central folder.

## Screenshots

![image](https://user-images.githubusercontent.com/17876556/151760848-4c45a258-e6fe-4325-9861-3b870668b885.png)

## Steps to Test

Follow the steps in `services/README.md` and verify that *both* services can start correctly. (You'll need to make sure that both the logger and ANT+ script have their dependencies installed)
